src/components/CodeInput.tsx

```diff
@@ -12,19 +12,19 @@ interface Props {
   isError?: boolean;
 }
 
-export const CodeInput = ({ value, onTextChange, testID, isError = false }: Props) => {
-  const [_, setValue] = useState('');
+export const CodeInput: FC<Props> = ({ value, onTextChange, testID, isError = false }) => {
+  const setCodeValue = (code: string) => {
```

Comment:
will be good to  use "useCallback"

Reply:
what is the reason that this function should use it?

Reply:
nvm, approved

Reply:
thx:) https://kentcdodds.com/blog/usememo-and-usecallback based on this article the useCallback would more performance even worse in this case

Reply:
want to share this same link paste hire, cos see diff side this scenario, but said nvm, for not blocked this simple changes